presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDirectSelectiveStreamReader.java

```diff
@@ -470,6 +469,31 @@ private int evaluateFilterWithNull(int[] positions, int positionCount)
         return positionsIndex;
     }
 
+    private int testEmptyStrings(int[] positions, int positionCount)
+    {
+        if (nonDeterministicFilter) {
+            int positionsIndex = 0;
+            for (int i = 0; i < positionCount; i++) {
+                int position = positions[i];
+
+                if (filter.testBytes("".getBytes(), 0, 0)) {
```

Comment:
should it use new byte[0] or should this be in a constant ?

Reply:
This is how it is it written in another place in this file, e.g. readWithFilter:

```
                        assert length == 0;
                        if (filter.testBytes("".getBytes(), 0, 0)) {
```

I assume the compiler would optimize this anyway.